packages/react-native-web/src/exports/Image/__tests__/__snapshots__/index-test.js.snap

```diff
@@ -329,14 +329,14 @@ exports[`components/Image prop "style" removes other unsupported View styles 1`]
 >
   <div
     class="css-view-175oi2r r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw r-backgroundSize-4gszlv"
-    style="filter: url(#tint-57);"
+    style="filter: url(#tint-58);"
```

Comment:
These `tint` id changes seems to be, because of the mock moved to the `beforeEach` block of the test 

The same happens if I apply the change on the `master` branch